### PR TITLE
Mixmix/test sign

### PR DESCRIPTION
**Pull request by mixmix**
This test adds:
- a simple test copying signing in CLI
- `test.only` support (so you can run **just one test** safely!)
    - this should have been in it's own PR, but I rammed it in here because I suspected it wouldn't get merged unless it was snuck in. #reflections

### CODE REVIEW

**Comment by frankiebee** [comment]
Cli is stable now so lets loop back into here this week @mixmix

**Comment by mixmix** [comment]
@frankiebee this is Ready for Review!

File: tests/sign.test.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  /* Sign */
+  const msg = Buffer
+    .from('Hello world: new signature from entropy!')
+    .toString('hex')
+
+  const signature = await run(
+    'sign',
+    entropy.signWithAdaptersInOrder({
+      msg: { msg },
+      order: ['deviceKeyProxy', 'noop'],
```
Comment (frankiebee) [inline tests/sign.test.ts:49]: noop program is not part of the default register so this is not needed. `deviceKeyProxy` should be the only adapter used sense it's the "default". Not sure what expcted behavior should be here any thoughts?

**Review by frankiebee — CHANGES_REQUESTED** [review]

File: tests/sign.test.ts (hunk 1 above)
Comment (mixmix) [reply tests/sign.test.ts:49]: why does the test pass then?

Comment (mixmix) [reply tests/sign.test.ts]: DONE!

**Comment by mixmix** [comment]
I assume that generated a notification.... we shall see!

**Comment by mixmix** [comment]
@frankiebee please review

**Review by frankiebee — APPROVED** [review]
Love having more tests!

**Pull request merged**